tutor/templates/build/openedx/Dockerfile

```diff
@@ -89,6 +89,9 @@ ENV SETTINGS tutor.production
 # Entrypoint will fix permissions of all files and run commands as openedx
 ENTRYPOINT ["docker-entrypoint.sh"]
 
+# Copy gunicorn settings file
+COPY gunicorn_conf.py /openedx/gunicorn_conf.py
+
 # Run server
 EXPOSE 8000
-CMD gunicorn --name ${SERVICE_VARIANT} --bind=0.0.0.0:8000 --max-requests=1000 ${SERVICE_VARIANT}.wsgi:application
+CMD gunicorn -c file:/openedx/gunicorn_conf.py --name ${SERVICE_VARIANT} --bind=0.0.0.0:8000 --max-requests=1000 ${SERVICE_VARIANT}.wsgi:application
```

Comment:
It's interesting, I didn't know it was possible to use a python configuration file like this. http://docs.gunicorn.org/en/stable/configure.html#configuration-file

I like the fact that we keep the `bind` and `max-requests` parameters out of the configuration file: I think it makes it easier to find the information about the exposed port.

Reply:
In my opinion a good spot for the `bind` and `max-requests` parameters is the `GUNICORN_CMD_ARGS` environment variable. But we can't do that with the current gunicorn version, so for now I agree they look good in the `CMD` directive.